Altis_Life.Altis/Functions.hpp

```diff
@@ -21,6 +21,22 @@ class SpyGlass {
     };
 };
 
+class Tonic {
```

Comment:
This can change to something more sensible. Some of them are utilities, some are `life` related